cmd/consul-dataplane/main.go

```diff
@@ -102,7 +102,9 @@ func init() {
 
 	// Default is false, may be useful for debugging unexpected termination.
 	BoolVar(flags, &flagOpts.dataplaneConfig.Envoy.DumpEnvoyConfigOnExitEnabled, "dump-envoy-config-on-exit", "DP_DUMP_ENVOY_CONFIG_ON_EXIT", "Call the Envoy /config_dump endpoint during consul-dataplane controlled shutdown.")
-
```

Comment:
**issue:** Please move this up a few lines so that it is with the other graceful flags.  Also, please remove `//########`